### PR TITLE
feature/impl actions

### PR DESCRIPTION
**Pull request by 4x8Matrix**
Implements a working `hello-world` command that allows the backend to throw in a custom argument.

### CODE REVIEW

File: Src/Services/ActionService.lua [hunk 1]
```diff
@@ -38,25 +43,81 @@ function ActionService.RegisterActionAsync(self: ActionService, action: any)
...
+					if serverArgumentData.type ~= argumentData.Type then
+						self.Reporter:Warn(`Argument '{argumentName}' has invalid types! Dropping Action request!`)
+					
+						return
+					end
+
+					if
+						argumentData.Type == ArgumentType.String
+						or argumentData.Type == ArgumentType.Number
+						or argumentData.Type == ArgumentType.Boolean
```
Comment (ethndotsh) [inline Src/Services/ActionService.lua:169]: Are we still doing the player type thingy? Where we return the player data when type is player?

File: Src/Services/ActionService.lua [hunk 2]
```diff
@@ -53,7 +57,52 @@ end
 
 function ActionService.OnStart(self: ActionService)
 	MessageReceiveService.OnAction:Connect(function(packet: ActionPacket)
-		-- fixme: account for 'behaviour' key in packet
+		if packet.behaviour == ActionBehaviour.SelectedServers or packet.behaviour == ActionBehaviour.SelectedServer then
```
Comment (ethndotsh) [inline Src/Services/ActionService.lua:65]: If it is selected server then it has to claim. That's the difference between selected servers and selected server (singular) (i just remembered)

**Review by ethndotsh — CHANGES_REQUESTED** [review]

File: Src/Services/ActionService.lua (hunk 2 above)
Comment (ethndotsh) [reply Src/Services/ActionService.lua:65]: and place id vs place ids

File: Src/Services/ActionService.lua (hunk 1 above)
Comment (4x8Matrix) [reply Src/Services/ActionService.lua:169]: yes

File: Src/Services/ActionService.lua (hunk 2 above)
Comment (4x8Matrix) [reply Src/Services/ActionService.lua:65]: oh, ok

**Pull request merged**